<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbrirChamado</class>
 <widget class="QWidget" name="AbrirChamado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Abrir Chamado</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Data:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="data_abertura_chamado">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2013</year>
        <month>6</month>
        <day>20</day>
       </date>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="descricao_chamado"/>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="titulo_chamado"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Titulo:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Descri√ßao:</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="botao_abrir_chamado">
        <property name="text">
         <string>&amp;Ok</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>titulo_chamado</tabstop>
  <tabstop>descricao_chamado</tabstop>
  <tabstop>data_abertura_chamado</tabstop>
  <tabstop>botao_abrir_chamado</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
